script/glide.sh

```diff
@@ -49,7 +36,7 @@ if ! type glide-vc > /dev/null 2>&1; then
   exit 1
 fi
 
-if [[ ! -e "${CWD}/glide.yaml" ]]; then
+if [[ ! -e "glide.yaml" ]]; then
```

Comment:
@timoreimann I know bash can be a lot worse... But I find this argument quite misleading. I'm sorry but I would have preferred to keep our bash scripts as simple/small as possible. We all know that a complex shell script is not maintainable.